livekit-agents/livekit/agents/metrics/base.py

```diff
@@ -98,6 +98,31 @@ class PipelineVADMetrics(VADMetrics):
     pass
 
 
+@dataclass
+class MultiModalLLMError(Error):
+    type: str | None
+    reason: str | None = None
+    code: str | None = None
+    message: str | None = None
+
+
+@dataclass
+class MultiModalLLMMetrics(LLMMetrics):
+    @dataclass
+    class InputTokenDetails:
+        cached_tokens: int
+        text_tokens: int
+        audio_tokens: int
+
+    @dataclass
+    class OutputTokenDetails:
+        text_tokens: int
+        audio_tokens: int
+
+    input_token_details: InputTokenDetails
+    output_token_details: OutputTokenDetails
+
```

Comment:
Let's reuse LLMMetrics and add more fields to it

Reply:
So did you mean to add `input_token_details` and `output_token_details` to `LLMMetrics`?

Reply:
No, feel free to ignore, I thing creating a new dataclass is fine